### PR TITLE
add camel-jaxb into extensions

### LINKED ISSUE #786: JAXB dataformat support
This might work already, we are perhaps just missing a test.

### PR DESCRIPTION
**Pull request by mmelko**
fix #786

### CODE REVIEW

**Comment by ppalaga** [comment]
https://github.com/apache/camel-quarkus/pull/849 came first and it might have an impact on the current one.

**Comment by mmelko** [comment]
retest this please

File: integration-tests/jaxb/pom.xml [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+            <groupId>org.apache.camel.quarkus</groupId>
+            <artifactId>camel-quarkus-direct</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.camel.quarkus</groupId>
+            <artifactId>camel-quarkus-core-xml</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.camel.quarkus</groupId>
+            <artifactId>camel-quarkus-bean</artifactId>
```
[the remark this answers is not in the document]
Comment (mmelko) [reply integration-tests/jaxb/pom.xml]: This was required for some reason when I was using `<setBody><simple>${body....}` when I will remove xml routes I can delete it.

[the remark this answers is not in the document]
Comment (lburgazzoli) [reply integration-tests/jaxb/pom.xml]: yes it is required for the simple language when using OGNL like expressions

File: extensions/jaxb/runtime/pom.xml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.camel.quarkus</groupId>
+            <artifactId>camel-quarkus-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.camel</groupId>
+            <artifactId>camel-jaxb</artifactId>
+            </dependency>
```
Comment (ppalaga) [inline extensions/jaxb/runtime/pom.xml]: indent

File: integration-tests/jaxb/pom.xml [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.apache.camel.quarkus</groupId>
+        <artifactId>camel-quarkus-integration-tests</artifactId>
+        <version>1.1.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>camel-quarkus-integration-test-jaxb</artifactId>
+    <name>Camel Quarkus :: Integration Tests :: JAXB extension</name>
+    <description>Integration tests for Camel Quarkus JAXB extension extension</description>
```
Comment (ppalaga) [inline integration-tests/jaxb/pom.xml]:
```suggestion
    <description>Integration tests for Camel Quarkus JAXB extension</description>
```

File: integration-tests/jaxb/pom.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (ppalaga) [reply integration-tests/jaxb/pom.xml]: camel-quarkus-bean is still there. Can we remove it?

[the remark this answers is not in the document]
Comment (mmelko) [reply integration-tests/jaxb/pom.xml]: I moved route from xml to java dsl so I still need it for simple language in setbody.

**Pull request merged**